lighthouse-core/lib/sd-validation/jsonld-keyword-validator.js

```diff
@@ -41,7 +41,7 @@ module.exports = function validateJsonLD(json) {
     if (name.startsWith('@') && !VALID_KEYWORDS.has(name)) {
       errors.push({
         path: path.join('/'),
-        message: 'Unknown keyword',
+        message: 'Unknown keyword ' + name,
```

Comment:
should we put this in quotes or after a colon or something? this also makes we wonder what our i18n story is here 😬